Documentation/Packaging/MRTK_Modularization.md

```diff
@@ -0,0 +1,167 @@
+# Mixed Reality Toolkit Componentization
```

Comment:
According to new documentation guidelines (https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Documentation/DocumentationGuide.md), headlines could be changed to sentence case.
_"Use Sentence case for headlines. Ie. capitalize the first letter and names, but nothing else."_